independent-publisher-connectors/OK dokument/apiDefinition.swagger.json

```diff
@@ -0,0 +1,681 @@
+                  "title": "Code",
+                  "x-ms-visibility": "internal"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message",
+                  "title": "Message",
+                  "x-ms-visibility": "internal"
+                },
+                "result": {
+                  "type": "string",
+                  "description": "Signature Request ID",
+                  "title": "Signature Request ID",
+                  "x-ms-visibility": "important"
+                },
+                "statusCode": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "statusCode",
+                  "title": "StatusCode",
+                  "x-ms-visibility": "internal"
+                },
+                "url": {
+                  "type": "string",
+                  "description": "url",
+                  "title": "Signature Request URL",
+                  "x-ms-visibility": "important"
+                },
+                "success": {
+                  "type": "boolean",
+                  "description": "Return True or False",
+                  "title": "Success",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "x-ms-visibility": "internal"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "string",
+                  "description": "code",
+                  "title": "code",
+                  "x-ms-visibility": "important"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message",
+                  "title": "message",
+                  "x-ms-visibility": "important"
+                },
+                "result": {
+                  "type": "string",
+                  "description": "result",
+                  "title": "",
+                  "x-ms-visibility": "internal"
+                },
+                "statusCode": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "statusCode",
+                  "title": "",
+                  "x-ms-visibility": "internal"
+                },
+                "success": {
+                  "type": "boolean",
+                  "description": "success",
+                  "title": "success",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "x-ms-visibility": "internal"
+                }
+              }
+            }
+          },
+          "405": {
+            "description": "Method Not Allowed",
+            "schema": {}

```

Comment:
@SoliteaSlovensko empty schemas are not allowed, you can remove it if you don't need this schema. You can refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) from the documentation.